modules/ossm-about-concepts-kiali.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+// about/ossm-about-concepts-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-concepts-kiali_{context}"]
+= {KialiProduct}
+
+{KialiProduct} provides visibility into your service mesh by showing you the microservices in your service mesh, and how they are connected.
+
+Kiali provides observability into the Service Mesh running on OpenShift Container Platform. Kiali helps you define, validate, and observe your Istio service mesh. It helps you to understand the structure of your service mesh by inferring the topology, and also provides information about the health of your service mesh.
+
+Kiali provides an interactive graph view of your namespace in real time that provides visibility into features like circuit breakers, request rates, latency, and even graphs of traffic flows. Kiali offers insights about components at different levels, from Applications to Services and Workloads, and can display the interactions with contextual information and charts on the selected graph node or edge.
```

Comment:
First, it probably is not entirely accurate to say "real time" -- I would say "near real time".
Second, I think `namespace` here should at least be plural (the graph view shows more than just one namespace); I would even consider saying "your mesh namespaces":

```suggestion
Kiali provides an interactive graph view of your mesh namespaces in near real time that provides visibility into features like circuit breakers, request rates, latency, and even graphs of traffic flows. Kiali offers insights about components at different levels, from Applications to Services and Workloads, and can display the interactions with contextual information and charts on the selected graph node or edge.
```